src/ConnectedMachine/ConnectedMachine.Autorest/examples/Get-AzConnectedNetworkSecurityPerimeterConfiguration.md

```diff
@@ -0,0 +1,22 @@
+### Example 1: {{ Add title here }}
```

Comment:
Please update this md file.
Fill in all placeholders.

Reply:
removed from this release